tests/test_api.py

```diff
@@ -16,7 +16,7 @@ def test_package_metadata() -> None:
 
 def test_method_result_is_str(func: str) -> None:
     method = getattr(platformdirs, func)
-    result = method("MyApp", "MyCompany")
+    result = method()
```

Comment:
seems unrelated change

Reply:
I was asked to remove the arguments from `get_user_documents_dir`, since it doesn't use them. The tests would fail here without the change.